src/main/java/newsonthego/Parser.java

```diff
@@ -15,7 +17,8 @@ public static void handleCommand(String command, String line, List<NewsArticle>
             NewsOnTheGo.getNews(line, list);
             break;
         case TOPICS:
-            NewsOnTheGo.showTopics();
+            // Assuming getNewsTopics() is a static method that returns a List<NewsTopic>
```

Comment:
Should we move this comment to somewhere else instead? I feel like it belongs in your main function.